README.md

```diff
@@ -46,8 +46,7 @@ obs.RequestName({args}, callback(err, data)) returns Promise
 obs.requestName({args}, callback(err, data)) returns Promise
 
 // The following are additional supported requests.
-obs.connect({ address: 'address' }, callback(err, data)) returns Promise
-obs.authenticate({ password: 'password' }, callback(err, data)) returns Promise
+obs.connect({ address: 'address', password: 'password' }, callback(err, data)) returns Promise
```

Comment:
My thought is that we should structure the API such that the consumer never has to worry about `_authenticate`. As far as they're concerned, there's just one `connect` method that does everything. The API might not be solid enough to actually get away with this plan yet, but that's the goal.